### PR TITLE
py: set default sample param to the right type

### PR DESCRIPTION
**Pull request by pongad**

### CODE REVIEW

File: src/test/java/com/google/api/codegen/testdata/py/python_library.baseline [hunk 1]
```diff
@@ -4409,26 +4409,28 @@ import sys
...
 
   client = library_v1.LibraryServiceClient()
 
   # name = 'Math'
+  # edition = 123
 
   if isinstance(name, six.binary_type):
     name = name.decode('utf-8')
 
+
```
Comment (vchudnov-g) [inline src/test/java/com/google/api/codegen/testdata/py/python_library.baseline]: Remove extra newline

File: src/main/resources/com/google/api/codegen/py/standalone_sample.snip [hunk 2]
```diff
@@ -115,6 +115,17 @@
   @end
 @end
 
+@private paramDefault(param)
```
Comment (vchudnov-g) [inline src/main/resources/com/google/api/codegen/py/standalone_sample.snip]: The default value should be what's specified in the config file. For example, `edition` defaults to `123`.

Were you planning to do this in a subsequent PR? If so, add a `# FIXME` here

File: src/main/resources/com/google/api/codegen/py/standalone_sample.snip [hunk 3]
```diff
@@ -81,7 +81,7 @@
     # default values should be correct type as well.
     parser = argparse.ArgumentParser()
     @join param : sample.initCode.argDefaultParams
-      parser.add_argument('--{@param.identifier}', default='')
+      parser.add_argument('--{@param.identifier}', type={@param.typeName}, default={@paramDefault(param)})
```
Comment (vchudnov-g) [inline src/main/resources/com/google/api/codegen/py/standalone_sample.snip]: Update the `# FIXME` above (but see my comment below about defaults)

File: src/test/java/com/google/api/codegen/testdata/py/python_library.baseline (hunk 1 above)
Comment (pongad) [reply src/test/java/com/google/api/codegen/testdata/py/python_library.baseline]: Done

File: src/main/resources/com/google/api/codegen/py/standalone_sample.snip (hunk 2 above)
Comment (pongad) [reply src/main/resources/com/google/api/codegen/py/standalone_sample.snip]: I didn't realize that's what we want. That's easy.

**Comment by pongad** [comment]
@vchudnov-g PTAL

File: src/main/resources/com/google/api/codegen/py/standalone_sample.snip [hunk 4]
```diff
@@ -81,7 +81,7 @@
     # default values should be correct type as well.
     parser = argparse.ArgumentParser()
     @join param : sample.initCode.argDefaultParams
-      parser.add_argument('--{@param.identifier}', type={@param.typeName}, default={@paramDefault(param)})
+      parser.add_argument('--{@param.identifier}', type={@param.typeName}, default={@renderInitValue(param.initValue)})
```
Comment (vchudnov-g) [inline src/main/resources/com/google/api/codegen/py/standalone_sample.snip]: Remove the `FIXME` above

File: src/main/resources/com/google/api/codegen/py/standalone_sample.snip (hunk 3 above)
Comment (vchudnov-g) [reply src/main/resources/com/google/api/codegen/py/standalone_sample.snip]: ping

File: src/main/resources/com/google/api/codegen/py/standalone_sample.snip (hunk 4 above)
Comment (pongad) [reply src/main/resources/com/google/api/codegen/py/standalone_sample.snip]: Done

**Pull request merged**